client/my-sites/checkout/controller.jsx

```diff
@@ -149,17 +149,15 @@ export function conciergeQuickstartSession( context, next ) {
 	);
 
 	const state = context.store.getState();
-	const selectedSite = getSelectedSite( state );
-
-	if ( ! selectedSite ) {
-		return null;
-	}
```

Comment:
The same case as with `ConciergeSessionNudge`: the check is not needed and `selectedSiteId` can be connected in the component itself.